plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveFileMetastore.java

```diff
@@ -85,4 +87,58 @@ public void testInsertOverwriteUnpartitioned()
     {
         // FileHiveMetastore has various incompatibilities
     }
+
```

Comment:
Is the scenario here easy to test in product test TestHiveTableStatistics ? That will make it more obvious what's the scenario we're tackling in this PR.